index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
 that are often, but not required to be, related.
         </p>
 
-      </section>
+    </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+This section seeks to illuminate the lifecycle of a credential. Credentials 
+allow one to make statements about a subject. Moreso, it provides a succint 
+composition mechanism for tamper-evident claims that provide cryptographic 
```

Comment:
Overly flowery language, no need to embellish. This section is also re-hashing concepts introduced elsewhere in the document, I think? Unless, this is a leading section, can't tell from the line numbers.